src/Microsoft.AspNet.Security.OpenIDConnect/Notifications/AuthorizationCodeReceivedNotification.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Security.OpenIdConnect;
+using Microsoft.IdentityModel.Protocols;
+using System.Diagnostics.CodeAnalysis;
+using System.IdentityModel.Tokens;
+
+namespace Microsoft.AspNet.Security.Notifications
```

Comment:
The other security providers use the `Microsoft.AspNet.Security`.`[Provider]` pattern:

https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Security.Cookies/Notifications/CookieApplyRedirectContext.cs#L9
https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Security.OAuth/Notifications/OAuthAuthenticatedContext.cs#L12

Definitely not a big deal, but something you should fix to improve consistency.